cmd/logger.go

```diff
@@ -154,36 +150,45 @@ func logIf(level logrus.Level, source string, err error, msg string, data ...int
 	if err == nil || isErrIgnored(err) {
 		return
 	}
-
-	fields := logrus.Fields{
-		"source": source,
-		"cause":  err.Error(),
-	}
-
-	if terr, ok := err.(*errors.Error); ok {
-		fields["stack"] = strings.Join(terr.Stack(), " ")
+	var trace []string
+	cause := err.Error()
+	// Get trace trace
+	trace = getTrace(3)
+	// Get time
+	time := time.Now().UTC().Format(time.RFC3339Nano)
```

Comment:
You would be getting `lint` error here because of shadowing `time` package.

Reply:
Also use UTCNow() helper instea

Reply:
Also use UTCNow() helper instead

Reply:
@balamurugana : Good point! Thank you.
Although I didn't get a lint error for it, I think it is a good idea to change the name to something other than `time`.
@harshavardhana : I've replaced `time.Now().UTC()` with `UTCNow()`.
So, it looks like `timeOfError := UTCNow().Format(time.RFC3339Nano)` right now.